examples/Kaleidoscope/run.jl

```diff
@@ -0,0 +1,51 @@
+function generate_IR(str)
+    cg = CodeGen()
+    ps = Parser(str)
+    while true
+        tok = current_token(ps)
+        if tok.kind == Kinds.EOF
+            break
+        else
+            r = ParseTopLevelExpr(ps)
+            codegen(cg, r)
+        end
+    end
+    LLVM.verify(cg.mod)
+    #LLVM.dispose(cg.ctx)
```

Comment:
Uncommenting this makes the tests segfault. I'm guessing one of these also disposes of the Module.